src/runtime/virtcontainers/qemu.go

```diff
@@ -298,6 +298,12 @@ func (q *qemu) setup(ctx context.Context, id string, hypervisorConfig *Hyperviso
 		q.arch.disableVhostNet()
 	}
 
+	if q.config.MemEncrypt == true {
+		q.arch.enableMemEncrypt()
+	} else {
+		q.arch.disableMemEncrypt()
+	}
+
```

Comment:
As noted in my review on the govmm code, I think you want to use the newer `confidential-guest-support` option, and update the naming of things accordingly.